scripts/pre_commit_linter.py

```diff
@@ -96,18 +95,25 @@ def _lint_js_files(node_path, jscs_path, autofix, config_jscsrc):
 
     print '----------------------------------------'
 
-    if not changed_js_filenames:
-        print 'There are no JavaScript files to lint in this commit. Exiting.'
-        print '----------------------------------------'
-        sys.exit(0)
+    if input_dir:
+        files_to_lint  = filter(_is_javascript_file, os.listdir(input_dir))
+        if not files_to_lint:
+            print 'There are no JavaScript files to lint in this folder. Exiting.'
+            print '----------------------------------------'
+            sys.exit(0)        
+    else:
+        if changed_js_filenames:
```

Comment:
It probably makes sense to do all the changed_filenames etc. computations in here instead, since they aren't relevant for the 'directory' case.